packages/anoncreds/src/repository/AnonCredsRevocationRegistryDefinitionPrivateRecord.ts

```diff
@@ -0,0 +1,62 @@
+import type { TagsBase } from '@aries-framework/core'
+
+import { BaseRecord, utils } from '@aries-framework/core'
+
+export enum RevocationRegistryState {
+  Created = 'created',
+  Active = 'active',
+  Full = 'full',
+}
+
+export interface AnonCredsRevocationRegistryDefinitionPrivateRecordProps {
+  id?: string
+  revocationRegistryDefinitionId: string
+  credentialDefinitionId: string
+  value: Record<string, unknown>
+  index?: number
```

Comment:
Is this the current index? Maybe we can call it `currentRevocationIndex` or something to make it a bit more descriptive?